icefall/dataset/asr_datamodule.py

```diff
@@ -0,0 +1,248 @@
+import argparse
```

Comment:
Regarding this file, since I added it to snowfall and used it I realized it’s a bit limiting for experimentation with the data pipeline. If you look at my GigaSpeech recipe, you’ll notice I made a local copy of this file in the recipe directory, modified it accordingly, and the train/decode scripts import it locally like this `from asr_datamodule import GigaSpeechAsrDataModule`. I think this design is much more flexible because if you need drastic changes you can copy this around or even make multiple versions.